webapp/backend/src/main/resources/data.sql

```diff
@@ -0,0 +1,34 @@
+INSERT INTO users (username, email, password)
+VALUES ('pan.doktor@pg.edu.pl', 'pan.doktor@pg.edu.pl', '123'),
+       ('admin', 'admin', '123');
+
+INSERT INTO authorities (username, authority)
+VALUES ('admin', 'ROLE_ADMIN'),
+       ('pan.doktor@pg.edu.pl', 'ROLE_USER');
+
+
+INSERT INTO SUBJECT (name, description, user_id)
+VALUES ('MBI-2022', 'Metody badawcze w informatyce', 'pan.doktor@pg.edu.pl'),
+       ('SOWW-2022', 'Systemy o wysokiej wydajności', 'pan.doktor@pg.edu.pl'),
+       ('BS-2022', 'Bezpieczeństwo systemów', 'pan.doktor@pg.edu.pl'),
+       ('BO-2022', 'Badania operacyjne', 'pan.doktor@pg.edu.pl'),
+       ('BS-2022', 'Biblioteki cyfrowe', 'pan.doktor@pg.edu.pl'),
+       ('UO-2022', 'Użyteczność oprogramowania', 'pan.doktor@pg.edu.pl');
```

Comment:
Was it tested against multiple `users`? Can we already have subjects owned by two different `users`?

Reply:
Question to @MateuszKepczynski